Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'warbler', platforms: :jruby
+gem 'warbler', git: 'https://github.com/jruby/warbler' if RUBY_PLATFORM == 'java'
```

Comment:
Why use this direct URL?

Reply:
The latest gem release of warbler is sadly long ago, but master contains fixes we need for the build.